### ISSUE TITLE
 Install-Package : Unable to load the service index for source https://dotnet.myget.org/F/roslyn/api/v3/index.json.

### ISSUE THREAD

**Issue by sabuncu** [emgucv/emgucv#396]
I am on Visual Studio 2019, latest version.

When I do  `Install-Package Emgu.CV -Version 4.4.0.4099`, I get the following error, and the package does not install.
>  
> Attempting to gather dependency information for package 'Emgu.CV.4.4.0.4099' with respect to project 'EmguCvHelloWorld', targeting '.NETFramework,Version=v4.7.2'
> Install-Package : Unable to load the service index for source https://dotnet.myget.org/F/roslyn/api/v3/index.json.
> Response status code does not indicate success: 404 (Feed does not exist.).
> At line:1 char:1
> Install-Package Emgu.CV -Version 4.4.0.4099
> 
>     CategoryInfo          : NotSpecified: (:) [Install-Package], Exception
>     FullyQualifiedErrorId : 
> 
> NuGetCmdletUnhandledException,NuGet.PackageManagement.PowerShellCmdlets.InstallPackageCommand
> 

I tried downloading the package first locally and installing it, but got the same error.

When I try to access the myget URL, I get a 'Feed does not exist' message in the browser.

Is this a known problem, and if so, what is the workaround?  Thanks.

Note: Previous package releases also display the same error.

**Comment by emgucv** [comment]
For 4.4 release, the Emgu.CV nuget package (shared between open source and commercial) is hosted on nuget.org, not myget.org

Only the commercial release runtime packages, (Emgu.CV.runtime.windows.dldt) are hosted on our commercial nuget server (myget.org). If you install the commercial runtime package, Visual Studio will also install the Emgu.CV nuget pacakge from nuget.org.

**Comment by sabuncu** [comment]
Thank you for the response, I appreciate it.

In the meantime, I was able to fix the problem as follows:

Tools -> NuGet Package Manager -> Package Manager Settings

Package Sources: I removed the package source entry for myget.

I am not sure how the myget entry made its way into the package sources on my Visual Studio 2019 installation.

Currently, I have two entries for package sources: nuget.org and Microsoft Visual Studio Offline Packages, in that order.

So, removing the myget package source entry fixes this problem.  I hope this will help anyone else who faces the same problem.
